guide/content/stylesheets/application.scss

```diff
@@ -104,3 +104,77 @@ $govuk-inverse-button-shadow-colour: govuk-shade($govuk-inverse-button-text-colo
     }
   }
 }
+
+@include govuk-exports("govuk/component/task-list") {
```

Comment:
Is all this CSS just temporary, so that this can be tested before the task list component gets shipped with govuk-frontend?

Reply:
Yeah, I [left a reminder](https://github.com/x-govuk/govuk-components/pull/445/commits/5872342e0a3cc5f22bb595593c38093f7a54c781) - I added an item to the checklist above so we don't forget.

Reply:
@peteryates I've merged the v5 branch into this branch and have removed this css.